.env

```diff
@@ -132,3 +132,6 @@ ESPRESSO_SEQUENCER_FETCH_RATE_LIMIT=25
 
 # Query service stress test
 ESPRESSO_NASTY_CLIENT_PORT=24011
+
+# Set this to the number of blocks you would like to confirm the sequencer can reach
+SMOKE_TEST_EXPECTED_BLOCK_HEIGHT=1
```

Comment:
why is this 1? Can we set it to a higher number like 300/500?

Reply:
So on anvil we get a block / second, right? so 300 blocks should be reasonable.

Reply:
dee59d4de40d1ac9d0e281b4b957acf6a22f76bc

Reply:
After some thinking I think `300` is to much for CI. I've set at `10` for now. Its configurable so we can move that locally if we want.